### PR TITLE
remove white space, adjust mobile view

### PR DESCRIPTION
**Pull request by Xeena-Newman**
Before shot of the mobile application view: 
Mobile_UI_Before

After shot of the mobile application view: 

Mobile_UI_After

### CODE REVIEW

File: src/assets/Timecard.scss [hunk 1]
```diff
@@ -3,3 +3,61 @@
     margin-top: 20px;
```
Comment (JWeston-HO) [inline src/assets/Timecard.scss:3]: Could you provide screenshots in the description. Demonstrating the changes made for this PR

Comment (IanFisherHO) [reply src/assets/Timecard.scss:3]: Description updated with before and after screenshots

**Review by JWeston-HO — DISMISSED** [review]

**Review by LaurieBennett — DISMISSED** [review]

File: src/assets/Timecard.scss
Comment (joannafawl) [inline src/assets/Timecard.scss:1]: Can we delete the `.responsive-margin` class now that it's no longer being used?

**Review by joannafawl — CHANGES_REQUESTED** [review]

File: src/components/common/form/message-summary/MessageSummary.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import MessageItem from '../message-item/MessageItem';
 const MessageSummary = ({ keys }) => {
   return (
     <div
-      className="govuk-notification-banner"
+      className="govuk-notification-banner mobile-banner"
```
Comment (stephenkaye-ho) [inline src/components/common/form/message-summary/MessageSummary.jsx:7]: Why have we added these classes? This isn't a div that only shows on mobiles. Is the media targeting of the screen size not enough to target the div

**Review by stephenkaye-ho — CHANGES_REQUESTED** [review]

File: src/assets/Timecard.scss [hunk 3]
```diff
@@ -3,3 +3,61 @@
...
   }
 }
+
+@media only screen and (max-width: 640px) {
+  .mobile-link-group {
```
Comment (stephenkaye-ho) [inline src/assets/Timecard.scss:8]: This styling is used to correct the nav links and can actually be applied regardless of the screen dimensions. I'd suggest giving it more semantic name moving it outside of the media rule

File: src/assets/Timecard.scss [hunk 4]
```diff
@@ -3,3 +3,61 @@
...
+  .govuk-caption-m {
+    margin-top: 2%;
+  }
+
+  .mobile-checkbox {
+    margin-top: -5%;
+  }
+
+  .mobile-heading {
+    margin-top: 0%;
```
Comment (stephenkaye-ho) [inline src/assets/Timecard.scss:24]: This only seems to appear next to the use of `govuk-caption-m` which is only used once. Why would we not just update this  `govuk-caption-m` style

File: src/assets/Timecard.scss
Comment (IanFisherHO) [reply src/assets/Timecard.scss:1]: This has now been removed

File: src/assets/Timecard.scss [hunk 5]
```diff
@@ -1,5 +1,70 @@
...
+}
+
+/* stylelint-disable selector-class-pattern */
+.govuk-summary-list__row {
+  display: flex;
+  align-items: center;
+  margin-bottom: 0;
+  padding: 17px 0;
+
+  /* disabling rule due to Gov not using standard style class names */
```
Comment (stephenkaye-ho) [inline src/assets/Timecard.scss:22]: This comment belong with the one above where styelint is  disabled

File: src/components/common/form/checkbox/Checkbox.jsx [hunk 6]
```diff
@@ -7,7 +7,7 @@ const Checkbox = ({ text, name, isChecked, handleChange }) => {
         <legend className="govuk-fieldset__legend govuk-fieldset__legend--l"></legend>
 
         <div className="govuk-checkboxes" data-module="govuk-checkboxes">
-          <div className="govuk-checkboxes__item">
+          <div className="govuk-checkboxes__item ">
```
Comment (stephenkaye-ho) [inline src/components/common/form/checkbox/Checkbox.jsx:10]: just need to remove the trailing space introduced

**Review by stephenkaye-ho — APPROVED** [review]

**Review by joannafawl — APPROVED** [review]

**Pull request merged**